tools/generate_conda_file.py

```diff
@@ -87,7 +87,7 @@
     "pymanopt": "pymanopt==0.2.5",
     "xlearn": "xlearn==0.40a1",
     "transformers": "transformers==2.5.0",
-    "tensorflow": "tensorflow==1.15.2",
```

Comment:
My understanding from the conversation we had was that we will upgrade to 1.15.4 to avoid the warning and later we will discuss how to move the code to TF2, @dciborow is this accurate?

Reply:
yes i have switched to 1.15.4